integration-tests/bats/commit.bats

```diff
@@ -42,6 +42,46 @@ teardown() {
     [[ "$output" =~ "t2" ]] || false
 }
 
+@test "commit: --all (-a) adds existing, changed tables" {
```

Comment:
Update `dolt/integration-tests/bats/helper/local-remote.bash` and remove the commit.bats line. This will ensure your changes run in both modes.